### ISSUE TITLE
Why make vggt and pi3 more powerful using mapanything framework

### ISSUE THREAD

**Issue by 7yzx** [facebookresearch/map-anything#111]
This is a really impressive piece of work, and thank you for open-sourcing the code and models.

I notice the finetuning results on pi3 are far more powerful than the original ones. Could you explain in detail why this training pipeline can improve the performance of other models?

I am new to finetuning and looking forward to your reply.By the way, do you have any plan to release the finetuning PI3/VGGT models?

**Comment by 7yzx** [comment]
Btw, How many dataset finetuning on pi3 can get a better result? 
Looking forward to your reply

**Comment by Nik-V9** [comment]
Hi, the presented finetuning results in the README use all the MapAnything datasets.

I believe the improvement stems from our covisibility guided view sampling, robust loss function (things like the exclude top N%, adaptive robust regression, etc) and relatively higher quality of data that we use.

**Comment by Nik-V9** [comment]
Closing issue, please reopen if further assistance is needed.
